invenio_vocabularies/contrib/common/ror/datastreams.py

```diff
@@ -30,7 +34,8 @@ def __init__(self, origin=None, mode="r", since=None, *args, **kwargs):
 
     def _iter(self, fp, *args, **kwargs):
         raise NotImplementedError(
-            "RORHTTPReader downloads one file and therefore does not iterate through items"
+            "RORHTTPReader downloads one file "
+            "and therefore does not iterate through items"
```

Comment:
Why is there so many formatting changes?

Reply:
I think the checks were not run in the past

Reply:
CI has been [running green recently](https://github.com/inveniosoftware/invenio-vocabularies/actions/runs/11189953992).

I just checked with my local setup and I don't get the same reformatting as you:

```bash
$ black --version
black, 24.10.0 (compiled: yes)
Python (CPython) 3.9.18

$ black .
All done! ✨ 🍰 ✨
215 files left unchanged.
```

Are you using another version of `black`, or another tool maybe?

Reply:
mine is 24.8 but since the major is the same, I wouldn't expect breaking changes. I guess it is not a blocker to merge?